llama_stack/distribution/build_container.sh

```diff
@@ -328,7 +344,8 @@ $CONTAINER_BINARY build \
   "$BUILD_CONTEXT_DIR"
 
 # clean up tmp/configs
-rm -f "$BUILD_CONTEXT_DIR/run.yaml"
+rm -f "$BUILD_CONTEXT_DIR/run.yaml" "$BUILD_CONTEXT_DIR/run-with-postgres-store.yaml"
```

Comment:
`run-with-postgres-store.yaml` isn't copied to $BUILD_CONTEXT_DIR by name above it is found with a find command, removing it by name here runs the risk of leaving behind other files that the find command matches.